controller/src/main/java/org/jboss/as/controller/StringListAttributeDefinition.java

```diff
@@ -17,9 +17,9 @@
 /**
  * @author <a href="mailto:tomaz.cerar@redhat.com">Tomaz Cerar</a>
  */
-public final class StringListAttributeDefinition extends PrimitiveListAttributeDefinition {
+public class StringListAttributeDefinition extends PrimitiveListAttributeDefinition {
```

Comment:
I originally modified this with the intention of subclassing it, but later resorted to extending PrimitiveListAttributeDefinition since it was easier to extend that class's builder.  However, CapabilityReferenceListAttributeDefinition can still subclass StringListAttributeDefinition, even if its builder does not.  I will sort that out.

Reply:
CapabilityReferenceListAttributeDefinition now extends StringListAttributeDefinition.